File: cadastro/WebApplication1/Model/pessoa.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace WebApplication1.Model;

public class pessoa
{
    public int Id { get; set; }
    public string TipoPessoa { get; set; }
    public string? NomeRazaoSocial { get; set; }
    public string? CpfCnpj { get; set; }
    public string Telefone { get; set; }
    public string? Email { get; set; }
    public virtual ICollection<endereco> Enderecos { get; set; }
}

File: cadastro/WebApplication1/Front/cadastro-api/src/app/shared/model/pessoa.ts
import { Endereco } from "./endereco";

export class Pessoa {
    constructor(){}

    public id: number;
    public tipoPessoa?: string;
    public nomeRazaoSocial?: string;
    public cpfCnpj?: string;
    public telefone?: string;
    public email?: string;
    public enderecos?: Endereco[]
}
